framework/src/application/storage/migrations/node/20200229000000_remove_signature_signSignature_from_transactions.sql

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+
+/*
+  DESCRIPTION: Remove signature and signSignature from trs
+*/
+
+ALTER TABLE trs
+DROP COLUMN "signature",
+DROP COLUMN "signSignature";
```

Comment:
I think this needs to be added to the list of things to be migrated when going to main net?

Reply:
I think this is covered under the migration issue. do we need anything extra to be done here?

Reply:
Ok if it's covered. Resolving then.